server/handlers/default.go

```diff
@@ -245,6 +245,75 @@ func getKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request,
 	return nil
 }
 
+// RotateKeyHandler rotates the remote key for the specified role, returning the public key
+func RotateKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	defer r.Body.Close()
+	vars := mux.Vars(r)
+	return rotateKeyHandler(ctx, w, r, vars)
+}
+
+func rotateKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
```

Comment:
I put this on the wrong line, so github hasn't collapsed this comment.  It has been addressed.  :)